src/main/java/pl/javastart/task/Television.java

```diff
@@ -0,0 +1,23 @@
+package pl.javastart.task;
+
+public class Television {
+    boolean televisionOn;
+
+    void turnOn() {
+        televisionOn = true;
+        System.out.println("Telewizor został włączony");
+    }
+
+    void turnOff() {
+        televisionOn = !televisionOn;
```

Comment:
Jeśli wywołam turnOff 2 razy, to znowu się włączy. Ma się po prostu wyłączyć, a jeśli jest wyłączony to ma taki pozostać.